styles/app.scss

```diff
@@ -5,4 +5,141 @@ $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 @import 'font-awesome/scss/font-awesome';
 @import 'box-sizing';
 
-@import 'adonis';
+body {
+  margin: 0;
+}
+
+
+//nav bar section
+.nav-bar {
+  display: flex;
+  max-width: 100%;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+
+  background-color: #962ab7;
+  color: white;
+}
+
+.nav-items  {
+  list-style-type: none;
+
+}
+
+//
+
+//navy box section
+
+.navy-box {
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+
+  background-color: #1b2977;
+  color: #eee;
+
+  min-height: 20rem;
+}
+
+//
+.container {
+  max-width: 100%;
+}
+
+//maroon section
+.maroon-box {
+  display: flex;
+
+  min-height: 20rem;
+
+  background-color: maroon;
+  color: white;
+}
+
+.maroon-text {
+  position: absolute;
+  top: 75%;
+  left: -5%;
```

Comment:
Look at the TIYO lesson on centering objects: https://online.theironyard.com/paths/579/units/3469/lessons/13952. Also remember that when you use `position: absolute`. You will want to make the "anchor" parent element `position: relative`: otherwise the element will be position in comparison to the full page (usually not what you want).